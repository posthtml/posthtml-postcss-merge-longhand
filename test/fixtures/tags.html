<div style="margin-top: 1px; margin-right: 2px; margin-bottom: 3px; margin-left: 4px;">test</div>
<p style="margin-top: 4px; margin-right: 3px; margin-bottom: 2px; margin-left: 1px;">test</p>
<section style="margin-top: 1px; margin-right: 2px; margin-bottom: 3px; margin-left: 4px;">test</section>
