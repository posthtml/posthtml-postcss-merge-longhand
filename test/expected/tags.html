<div style="margin: 1px 2px 3px 4px;">test</div>
<p style="margin: 4px 3px 2px 1px;">test</p>
<section style="margin-top: 1px; margin-right: 2px; margin-bottom: 3px; margin-left: 4px;">test</section>
